### PR TITLE
Fix gitlab pages generation

### PR DESCRIPTION
**Pull request by zabbal**
Revert previously requested renaming as it breaks pages generation.

### CODE REVIEW

**Comment by Raigi** [comment]
I guess you should rebase first: This branch is out-of-date with the base branch

**Comment by simao** [comment]
Please rebase, not merge.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -116,8 +116,8 @@ start deps scan:
     paths:
       - depchk.html
 
-generate pages:
-  stage: publish dependencies scan
+pages:
+  stage: pages
```
Comment (simao) [inline .gitlab-ci.yml]: Please use a better name for this, this is not the cause of this error, we use descriptive names for other jobs, see for example deployment descriptors.

Comment (alexhumphreys) [reply .gitlab-ci.yml]: https://docs.gitlab.com/ce/ci/yaml/README.html#pages I think the _stage_ can be renamed, but not the _job_.

Comment (simao) [reply .gitlab-ci.yml]: Ah! Ok, lets change the stage name then.

**Pull request merged**